examples/blockly-webpack/package.json

```diff
@@ -9,7 +9,7 @@
     "build": "webpack"
   },
   "dependencies": {
-    "blockly": "^3.20200123.1"
+    "blockly": "8.0.5"
```

Comment:
We should keep the `^` here, which means "any release at or above the one listed but below the next major version". That way if we ever release a patch for v8 this demo could use it automatically, and it's good practice to model for people copying this example

(we just launched v9, but this would need further changes to be compatible with v9, since we changed the way that you import generators, so i'm fine leaving this at v8 for now)